src/helm/config/model_deployments.yaml

```diff
@@ -200,6 +200,15 @@ model_deployments:
       class_name: "helm.benchmark.window_services.anthropic_window_service.AnthropicWindowService"
       args: {}
 
+  - name: anthropic/claude-2.1
```

Comment:
You need to do something like:

```
  - name: anthropic/claude-2.1
    model_name: anthropic/claude-2.1
    tokenizer_name: anthropic/claude
    max_sequence_length: 8000
    max_sequence_and_generated_tokens_length: 9016
    client_spec:
      class_name: "helm.proxy.clients.anthropic_client.AnthropicClient"
      args: {}
    window_service_spec:
      class_name: "helm.benchmark.window_services.anthropic_window_service.AnthropicWindowService"
      args: {}
```

Reply:
Thanks for catching this- you're right this won't work as is! My git tree had issues so I manually pasted these lines in and accidentally used the same lines twice. 

Now it's fixed and should be working.